core/trino-parser/src/main/java/io/trino/sql/SqlFormatter.java

```diff
@@ -650,6 +656,15 @@ protected Void visitQuery(Query node, Integer indent)
                     }
                     builder.append('\n');
                 }
+
+                Iterator<SessionSpecification> sessionProperties = node.getSessionProperties().iterator();
```

Comment:
This should go above functions. Are we missing some tests? Otherwise, this should fail.

Reply:
It's moved in a fixup commit.